db/migrate/20190722082648_add_derived_estimated_hours_to_work_packages.rb

```diff
@@ -29,18 +29,23 @@ def change
   # estimated hours set based on their children through the UpdateAncestorsService.
   #
   # We move this value to the derived_estimated_hours column and clear
-  # the estimated_hours column. In the future users can estimte the time
-  # for parent work pacages separately there while the UpdateAncestorsService
+  # the estimated_hours column. In the future users can estimate the time
+  # for parent work packages separately while the UpdateAncestorsService
   # only touches the derived_estimated_hours column.
   def migrate_to_derived_estimated_hours!
     last_id = Journal.order(id: :desc).limit(1).pluck(:id).first || 0
     wp_journals = "work_package_journals"
-    work_packages = WorkPackageWithRelations.with_children
 
+    work_packages = WorkPackageWithRelations.with_children.where("estimated_hours > ?", 0)
     work_packages.update_all("derived_estimated_hours = estimated_hours, estimated_hours = NULL")
```

Comment:
As you are not really interested in keeping the work_packages, it would IMO have been clearer to write
```
WorkPackageWithRelations
.with_children
.where("estimated_hours > ?", 0)
.update_all("derived_estimated_hours = estimated_hours, estimated_hours = NULL")
```

I stumbled over the code reusing the same variable for no apparent reason until I saw the `update_all`.